bigbluebutton-html5/imports/api/breakouts/server/methods/createBreakout.js

```diff
@@ -18,7 +18,7 @@ export default function createBreakoutRoom(credentials, rooms, durationInMinutes
   check(requesterToken, String);
 
   const eventName = 'CreateBreakoutRoomsCmdMsg';
-
+  if (rooms.length > 8) return Logger.info(`Attempt to create breakout rooms with invalid number of rooms in meeting id=${meetingId}`);
```

Comment:
Shouldn't the max number of rooms be a meeting property?

Reply:
I don't think so, from the beginning we decided to do it hard coded in the client, First version I did it in the settings, but was requested to change for hard code

Reply:
In https://github.com/bigbluebutton/bigbluebutton/pull/7507/files :

`There needs to be a hard cap of 9 breakout rooms or other parts of the application won't work. One problematic part would be the voice bridge generation.`